docs/meshstack.emergency-users.md

```diff
@@ -0,0 +1,98 @@
+---
+id: meshstack.emergency-users
+title: Emergency Users
+---
+
+meshStack manages access to cloud platforms, projects and resources. In cases when urgent intervention is required and users with regular access permissions are not available there must be a defined process to securely access projects and their associated platform resources. This page outlines step-by-step procedures that meshStack operators can use as the basis for their own emergency procedures. Depending on your organization's requirements, these procedures can be augmented with additional organisational or technical procedures.
+
+Example use cases for emergency users and emergency intervention include
+
+- An important application has stopped functioning and operating users needs access to debug and fix the problem.
+- Project access for a specific user must be immediately revoked (e.g. due to an account compromise).
+
+In all cases access permissions can always be modified through the [meshPartner](administration.index.md) account which is managed by an operations team. If available, a user with customer admin access is also sufficient for some cases.
+
+## Emergency Access with Customer Admin
+
+If a user with [customer admin](meshcloud.customer.md) access is available, project users and roles can be managed the normal way, even if the customer admin is not assigned to the project.
+
+First, the user requiring emergency access must be invited to the customer
+
+- Ensure that the correct customer is selected
+- Open "Account" from the settings menu in the top right
+- Navigate to "Users" and create an invitation
+  
+You can also grant customer admin rights to the newly invited user, i.e. if the emergency user needs to modify other user permissions. In this case, the new user can perform the following steps themselves.
+
+### Adding emergency as Customer Admin
+
+The user can then be assigned to projects belonging to the customer:
+
+- Open "Account" from the settings menu in the top right
+- Navigate to "Projects" and select "Edit Project" from the actions column for the correct project
+- Add user to desired project role
+
+Since emergency access should only be temporary, it's strongly advised to assign the user a role with a [set expiration date](./meshcloud.project.md#expiry-of-a-user-assignment) which will ensure that the user is automatically removed from the project after the specified date.
```

Comment:
```suggestion
Since emergency access should only be temporary, it's strongly advised to assign the user a role with a [set expiration date](./meshcloud.project.md#expiry-of-a-user-assignment) which will ensure that the user is automatically removed from the meshProject after the specified date.
```